### DISCUSSION TITLE
.live domain not working

### DISCUSSION THREAD

**Discussion by psnwd** [vercel/community#2909] [Help] [answered]
### Summary

I encountered an issue while attempting to add a .live domain and subdomain. Despite waiting for 48 hours, the mapping did not work as expected. Surprisingly, when I tried the same configuration with a different server, it functioned perfectly. I have confirmed that the Cloudflare configuration is correct. Additionally, I successfully added a .com subdomain to the project without any problems. I would greatly appreciate assistance in resolving this issue, as it seems to be specific to the .live domain and subdomain configuration. Thank you.

This `.com` sub domain working
http://site.sublaze.com/

### Example

http://cloudwind.live/

### Steps to Reproduce

1. Create project
2. Add `.live` domain (map with Cloudflare)
3. Then wait sometime and try to visit that URL

**Comment by smaeda-ks** [comment, marked as answer]
@psnwd could you define what exactly "not working" means in this case? Is this a redirect loop issue? If so, the issue appears to be in your Cloudflare configuration, not with Vercel.

```
# Cloudflare is serving 308 redirect to the "same" URL:
$ curl -svo /dev/null 'https://cloudwind.live/' 2>&1 | grep -i location
< location: https://cloudwind.live/
```

but if you send a request against Vercel directly, you won't get such a redirect:

```
# returns 200 response fine
$ curl -svo /dev/null 'https://cloudwind.live/' --connect-to cloudwind.live:443:cname.vercel-dns.com
```

so please double check your Cloudflare configuration around redirections

**Comment by psnwd** [comment]
OK, I'll double check that.
